ui/src/scenes/Schema/Table/index.tsx

```diff
@@ -111,65 +120,80 @@ const Table = ({
   const [columns, setColumns] = useState<QuestDB.Column[]>()
 
   useEffect(() => {
-    if (expanded) {
-      combineLatest(
-        from(quest.showColumns(name)).pipe(startWith(null)),
-        of(true).pipe(delay(1000), startWith(false)),
-      ).subscribe(([response, loading]) => {
-        if (response && response.type === QuestDB.Type.DQL) {
-          setColumns(response.data)
-          setLoading(false)
-        } else {
-          setLoading(loading)
-        }
-      })
-    }
-  }, [expanded, refresh, quest, name])
-
-  const handleClick = useCallback(
-    (e) => {
-      onChange(expanded ? "" : name)
+    combineLatest(
```

Comment:
ideally this should be removed, because `Tree` handles data loading through `onOpen`.

but i left it here because i don't want to break existing functionality with `refresh` and possibly other areas. didn't investigate much what removal of this would cause, so leaving for now

Reply:
Good thinking, let's add a comment in the code so that we don't forget why seemingly duplicating the fetch logic 🙂